modules/planning/common/path/path_data.h

```diff
@@ -94,24 +94,16 @@ class PathData {
     return blocking_obstacle_id_;
   }
 
-  const size_t trimmed_path_bound_size() const {
-    return trimmed_path_bound_size_;
-  }
-  void set_trimmed_path_bound_size(const size_t trimmed_path_bound_size);
-
-  const common::PointENU path_reference_end_pose() const {
-    return path_reference_end_pos_;
-  }
-  void set_path_reference_end_pose(
-      const common::PointENU &path_reference_end_pose);
-
   const bool is_valid_path_reference() const {
     return is_valid_path_reference_;
   }
   void set_valid_path_reference(bool is_valid_path_reference) {
```

Comment:
function name use: set_<member variable name>

Reply:
Can you be more specific? Am I not supposed to use set here?

Reply:
oh I previously typed "set_\<member_variable\>".  seems got truncated as marker. 
to be specific, use "set_is_valid_path_reference()"

Reply:
Fixed. Thank you.